src/Symfony/Component/DependencyInjection/Tests/Fixtures/graphviz/services9.dot

```diff
@@ -30,6 +30,9 @@ digraph sc {
   node_lazy_context [label="lazy_context\nLazyContext\n", shape=record, fillcolor="#eeeeee", style="filled"];
   node_lazy_context_ignore_invalid_ref [label="lazy_context_ignore_invalid_ref\nLazyContext\n", shape=record, fillcolor="#eeeeee", style="filled"];
   node_closure_proxy [label="closure_proxy\nBarClass\n", shape=record, fillcolor="#eeeeee", style="filled"];
+  node_bar [label="BAR\nstdClass\n", shape=record, fillcolor="#eeeeee", style="filled"];
+  node_bar2 [label="bar2\nstdClass\n", shape=record, fillcolor="#eeeeee", style="filled"];
+  node_bar2 [label="BAR2\nstdClass\n", shape=record, fillcolor="#eeeeee", style="filled"];
```

Comment:
Graphviz nodes seem to be case-sensitive. so removing the lowercase should fix this.